extensions/ql-vscode/src/databases/ui/db-panel.ts

```diff
@@ -290,4 +299,18 @@ export class DbPanel extends DisposableObject {
     // You can only select one item at a time, so selection[0] gives the selection
     return this.treeView.selection[0]?.dbItem;
   }
+
+  private async openOnGitHub(treeViewItem: DbTreeViewItem): Promise<void> {
+    if (treeViewItem.dbItem === undefined) {
+      throw new Error("Unable to open on GitHub. Please select a valid item.");
+    }
+    const githubUrl = getGitHubUrl(treeViewItem.dbItem);
+    if (!githubUrl) {
+      throw new Error(
+        "Unable to open on GitHub. Please select a remote repository or owner.",
+      );
+    }
+
+    await commands.executeCommand("vscode.open", Uri.parse(githubUrl));
```

Comment:
FYI, I haven't added any kind of validation for whether this is a valid link, so people will get a 404 if their repo or owner isn't accessible.

I think this is okay: we check that the repo/owner is of the right "shape" in the config, and then we check whether it's accessible when running a variant analysis.